<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagoEfectivo</class>
 <widget class="QMainWindow" name="pagoEfectivo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cargo a Libreta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Total Venta: $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/error.png</normaloff>../icon/error.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAceptar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/binoculars.png</normaloff>../icon/binoculars.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFin">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>150</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FIN</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icon/success.png</normaloff>../icon/success.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;N° Cliente : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTotalVenta">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>360</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;N° Boleta:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PAGO CARGO LIBRETA&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtNboleta">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtIdCliente">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombreCliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>641</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;**********************************************************&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdCupoDisponible">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>240</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdDeudaTotal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Cupo Disponible: $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Deuda Total: $&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tk">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticket  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblidVenta">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>******</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIdUsuario">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>****</string>
    </property>
   </widget>
   <widget class="QLabel" name="Usuario">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="displayTotalVenta">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">self.displayNumber.setStyleSheet(&quot;background-color: rgb(0, 255, 0)&quot;)
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string extracomment="$"/>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
